jwst/datamodels/schemas/ifucubepars.schema.yaml

```diff
@@ -1,6 +1,6 @@
 title: Default IFU CUBE parameters data model
 allOf:
-- $ref:	referencefile.schema.yaml
+- $ref: referencefile.schema.yaml
```

Comment:
Tabs are not considered valid by the YAML parser. Maybe there should be some kind of git hook to check for this in new commits?

Reply:
I would hope delinters would catch this, but requires one to actually use them.  One of our TravisCI tests actually runs flake8, but failures there do not prevent the code from being merged or show up as failures.  Maybe it should?

Reply:
Here's the flake8 test on TravisCI for the last commit on this PR.

https://travis-ci.org/spacetelescope/jwst/jobs/404451874